### PR TITLE
[Gfx11xx] fix build break

### PR DESCRIPTION
**Pull request by cmingch**
1. gfx11xx don't support preload_arg
2. build break when --architecture gfx11xx only
3. BB for ext ops with gfx11xx

### CODE REVIEW

File: tensilelite/Tensile/TensileInstructions/Code.py [hunk 1]
```diff
@@ -680,7 +680,7 @@ def __str__(self):
         kStr += kdIndent + ".amdhsa_system_vgpr_workitem_id %u\n" % self.vgprWorkItem
         kStr += kdIndent + ".amdhsa_float_denorm_mode_32 3\n"
         kStr += kdIndent + ".amdhsa_float_denorm_mode_16_64 3\n"
-        if self.enablePreloadKernArgs:
+        if self.enablePreloadKernArgs and (self.kernel.isa == (9, 0, 10) or self.kernel.isa == (9, 4)):
```
Comment (jichangjichang) [inline tensilelite/Tensile/TensileInstructions/Code.py]: Should disable "PreloadKernArgs" in SolutionStruct.py directly, but not here.

File: tensilelite/Tensile/Ops/SoftmaxGenerator.py [hunk 2]
```diff
@@ -605,6 +605,16 @@ def kernel_rodata(name: str):
 .amdhsa_next_free_vgpr .amdgcn.next_free_vgpr
 .amdhsa_next_free_sgpr .amdgcn.next_free_sgpr
 .end_amdhsa_kernel
+''' if gfx_arch >= (9, 0, 10) and gfx_arch < (10, 0, 0) else\
+f'''
+.rodata
```
Comment (jichangjichang) [inline tensilelite/Tensile/Ops/SoftmaxGenerator.py]: better not to produce redundent codes

File: tensilelite/Tensile/TensileInstructions/Code.py (hunk 1 above)
Comment (cmingch) [reply tensilelite/Tensile/TensileInstructions/Code.py]: done

File: tensilelite/Tensile/Ops/SoftmaxGenerator.py (hunk 2 above)
Comment (cmingch) [reply tensilelite/Tensile/Ops/SoftmaxGenerator.py]: done

**Review by jichangjichang — APPROVED** [review]
lgtm

**Review by hcman2 — APPROVED** [review]

**Pull request merged**